mjcf_to_sdformat/tests/test_defaults.py

```diff
@@ -58,10 +58,13 @@ def test_defaults(self):
         link = model.link_by_index(1)
         self.assertNotEqual(None, link)
         visual = link.visual_by_index(0)
-        self.assertEqual(sdf.GeometryType.SPHERE, visual.geometry().type())
-        sphere_shape = visual.geometry().sphere_shape()
-        self.assertNotEqual(None, sphere_shape)
-        self.assertEqual(0.01, sphere_shape.radius())
+        print(f"Link {link.name()} visual {visual.name()}")
```

Comment:
remove this print?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/61/commits/78a98a781aa40f8bb85a191e18717f3c490dbff4